Builds/CMake/CMakeFuncs.cmake

```diff
@@ -35,6 +35,7 @@ macro(parse_target)
       string(REGEX REPLACE "^\\.?[^\\.]+(.*$)" "\\1" remaining ${remaining})
 
       if (${cur_component} STREQUAL gcc)
+        set(expect_cc true)
```

Comment:
Q: I'm trying to figure out scenario `expect_cc` protects against. If we didn't have this, and unconditionally check if it's set (and not `-NOTFOUND`) before we set the cache, what bad thing happens?